spacy/errors.py

```diff
@@ -593,7 +595,12 @@ class Errors(object):
     E997 = ("Tokenizer special cases are not allowed to modify the text. "
             "This would map '{chunk}' to '{orth}' given token attributes "
             "'{token_attrs}'.")
- 
+    E998 = ("Could not match the gold entity links to entities in the doc - "
+            "make sure the gold EL data refers to valid results of the "
+            "named entity recognizer in the `nlp` pipeline.")
```

Comment:
@adrianeboyd : I think E998 has become obsolete - probably a left over from your rebase/remerge to `develop`?

Reply:
Yeah, I did it in the github editor and wanted to double-check, thanks!